Finish translation of "Party types" (#597)
Altinn/altinn-studio-docs: content/app/development/configuration/partytype/_index.en.md

description: How to ensure that only specific types of party types can use an app.
toc: true
weight: 150
---

> ## Limit access in Altinn Studio designer

Reviewer: ```suggestion
## Limit access using Altinn Studio designer
```
I suggest changing the headings a bit to make it clear that this approach and the one below are two different alternatives to reach the same goal.

> You can limit which actors have access to an app by using the Altinn Studio Designer. Role access options are placed in the "create" category, on the page "Tilgangsstyring", represented by the keyhole on the leftmost side menu.

Reviewer: ```suggestion
You can limit which actors have access to an app by using the Altinn Studio Designer. Role access options are placed in the "Lage" category in the top menu, on the page "Tilgangsstyring", represented by the keyhole on the leftmost side menu.
```

On this page you can mark the actor types the app should be accessible for. If none (or all) actor types are marked, everyone will have access. The option "Virksomhet" is used in cases where a report is filed on behalf of a company or other organization.

The screenshot below shows the settings for full access to the application for all organization numbers (both main entity and sub entities).

![GUI for controlling access between actor types](partytypeselector.png "GUI for controlling access between actor types")

> ## Adjusting applicationmetadata.json

Reviewer: ```suggestion
## Limit access by editing applicationmetadata.json
```
